### PR TITLE
[WIP] Update Jura citation style

### PR DESCRIPTION
**Pull request by zuphilip**
This update the citation style for Jura based on some feedback.

However, I would like to discuss the approach for advance types and especially the last commit, which looked like fixing a potential older issue.

CC @sebeka

### CODE REVIEW

File: Configuration/FlexForms/flexform_publist_pi1.xml [hunk 1]
```diff
@@ -300,15 +300,23 @@
          </numIndex>
          <numIndex index="14">
           <numIndex index="0">LLL:EXT:uma_publist/Resources/Private/Language/locallang_be.xlf:pi1_flexform.type.workshop_item</numIndex>
-          <numIndex index="1">adv:workshop</numIndex>
+          <numIndex index="1">workshop_item</numIndex>
```
Comment (zuphilip) [inline Configuration/FlexForms/flexform_publist_pi1.xml:303]: This fixes a problem with this option list by making sure that the names match to the ones defined everywhere else. Was there any reason for this prefix `adv` and different names used there?

**Review by sebeka — APPROVED** [review]
Why there is a space in front of ":" in titles? Comes it from madoc or from the plugin itself?
Anyway, I can live with that solution, Thanks!

File: Configuration/FlexForms/flexform_publist_pi1.xml (hunk 1 above)
Comment (sebeka) [reply Configuration/FlexForms/flexform_publist_pi1.xml:303]: I think I added "adv:" just to make clear, that this is an extra attribute, which is not supported by madoc.

Comment (zuphilip) [reply Configuration/FlexForms/flexform_publist_pi1.xml:303]: Okay, but the prefix `adv:` is only used here. Therfore, these options didn't have an effect for me. It only worked when addin `ALL` as well. For example, it was not possible for me to reorder the publication types including workshop items. @sebeka Does that work for you?

**Comment by zuphilip** [comment]
The ` : ` (including spaces around it) is used by librarians to separate the title from the sub-title and other similar cases. This is part of how to type this into MADOC, but we can change the display on websites in the plugin, as the first commit here is doing it for the law style.

**Pull request merged**